test/git_checkout_test.go

```diff
@@ -303,6 +310,10 @@ func TestGitPipelineRunFail_HTTPS_PROXY(t *testing.T) {
 // successfully under different revision inputs (default branch, branch)
 // This test will run on spring-petclinic repository which does not contain a master branch as the default branch
 func TestGitPipelineRunWithNonMasterBranch(t *testing.T) {
+	if IsExcluded(t.Name()) {
+		t.Skipf("skip for %s architecture", runtime.GOARCH)
+	}
+
```

Comment:
nit: As this is duplicated a bunch of times, this could even be a function : `skipIfExcluded(t.Name())`.

Reply:
good point! will do

Reply:
however it will be `skipIfExcluded(t)`, as we need `t` to skip it

Reply:
true :grin: